Makefile

```diff
@@ -14,13 +14,13 @@ tests: rpg cppcheck secondary_cppcheck valgrind outputs
 rpg: $(OBJS)
 	$(CC) $(CFLAGS) -o rpg $(OBJS)
 
-creature.o: creature.cpp
+creature.o: creature.cpp creature.h
 	$(CC) $(CFLAGS) -c creature.cpp
```

Comment:
Es a creature nem hasznalja monjduk a json-t?

Reply:
Hozzáírtam a JsonParser.h-t is.